plugin/wavesurfer.microphone.js

```diff
@@ -122,8 +122,32 @@
             this.disconnect();
 
             // stop stream from device
-            if (this.stream) {
-                this.stream.stop();
+<<<<<<< HEAD
+            if (this.stream && this.stream.getTracks()) {
```

Comment:
The check of the stream being not null should be outside of the `getTracks` change.

```
if (this.stream) {
    if (this.stream.getTracks) { // note that this should not be a call
        this.stream.getTracks().forEach(function (stream) {
            stream.stop();
        });
    } else {
        this.stream.stop():
    }
}
```